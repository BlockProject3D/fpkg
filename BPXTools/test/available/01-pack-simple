export NAME="Pack (SIMPLE)"
export COMMAND="-f test.bpx pack ../LICENSE.txt"
export DESC="Test the pack command"
export ADDITIONAL="diff test.bpx test/available/test.bpx"
export STATUS=0