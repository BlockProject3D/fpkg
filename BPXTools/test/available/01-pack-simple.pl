$Test = {
    Name => "Pack (SIMPLE)",
    Command => "-f test.bpx pack test/LICENSE.txt",
    Description => "Test the pack command",
    Status => 0
};

sub TestBegin {
    CRLFToLF("../LICENSE.txt", "test/LICENSE.txt");
}

sub TestEnd {
    return EnsureEqual("test.bpx", "test/available/test.bpx");
}
