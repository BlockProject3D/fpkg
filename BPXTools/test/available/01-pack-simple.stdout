Writing file LICENSE.txt with 1517 byte(s)
Writing section #0: Size = 12, Size after compression = 12
Writing section #1: Size = 1529, Size after compression = 1529
